screens/Post.js

```diff
@@ -32,6 +32,7 @@ export default function Post({ route, navigation }) {
     const dateInstance = new Date(article.date)
     const authors = article.parsely?.meta?.creator?.reduce((object, name, index) => ({...object, [name]: article.coauthors[index]}), {})
     const [displayCategory, setDisplayCategory] = useState({})
+    const [Caption, setCaption] = useState({})
```

Comment:
Let's rename `Caption` to `caption`. It doesn't really make a difference in terms of functionality, but in this context it could be mistaken as a component. Also, a caption is a string, so the empty dictionary literal should be an empty string instead.

Reply:
@MatthewTurk247 thank you for the feedback let me resolve it